### PR TITLE
README.md: Move developer info, streamline installation

### PR DESCRIPTION
**Pull request by bcipolli**
Title says it all. Not sure this is what @jnmarcus or @adborden wanted, but we can iterate on this. Mostly, wanted to beef up the installation information and say: yay, I got it working! :+1:

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
+# Guide to Contributing
+
+This Angular starter app is built with best practices in mind. The folder structure is intended to be different than the standard official AngularJS application. The application architecture follows modern design conventions from the proposal outlined here:
+
+https://docs.google.com/document/d/1XXMvReO8-Awi1EZXAXS4PzDzdNvV6pGcuaF4Q9821Es/pub
```
Comment (adborden) [inline Contributing.md]: IMHO I think we can ditch this doc and trim it down to just this link.

**Comment by adborden** [comment]
:+1:

**Comment by jnmarcus** [comment]
@adborden are you talking about the google docs link? or a link to the contributing.md?

**Comment by bcipolli** [comment]
I believe the google doc. I would be happy to make that change, does that work for you, @jnmarcus ?

**Comment by jnmarcus** [comment]
@bcipolli it's just that it's still a little off from the components architecture..I'm not even sure how up to date it is either since I'm not sure that's what Google is using anymore.  If anything I would prefer this: https://github.com/johnpapa/angular-styleguide#modularity, but to be honest I don't really see why we can't just keep contributing.md for now and deal with this later.  The goal for this pull request was to make the readme more readable - so that developers could get up and running faster, and I think that your original pull request already accomplished that.

**Comment by bcipolli** [comment]
Sure. @adborden would it be cool to keep the `contributing.md` for the moment, and discuss this further in a github issue on where/how to document the front-end component architecture?

**Comment by adborden** [comment]
Sure, let's merge this, replace the google doc with the angular-styleguide link.

**Pull request merged**